### PR TITLE
Dbyrdaquent seo working branch

### PR DESCRIPTION
**Pull request by dbyrdaquent**
@DaniHalfin

- ready for proof and merge

### CODE REVIEW

File: microsoft-365/compliance/GDPR-discovery-protection-reporting-in-Office365-dev-test-environment.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "GDPR discovery, protection, and reporting in the Office 365 dev/test environment"
-description: "Demonstrate GDPR capabilities in Office 365."
+description: Learn how to configure and demonstrate personally identifiable information (PII) for GDPR an in Office 365 dev/test environment.
```
Comment (DaniHalfin) [inline microsoft-365/compliance/GDPR-discovery-protection-reporting-in-Office365-dev-test-environment.md]: typo here?

Comment (dbyrdaquent) [reply microsoft-365/compliance/GDPR-discovery-protection-reporting-in-Office365-dev-test-environment.md]: Yes. Definitely. I will fix.

Comment (dbyrdaquent) [reply microsoft-365/compliance/GDPR-discovery-protection-reporting-in-Office365-dev-test-environment.md]: Corrected and re-pushed. Thank you.

**Comment by dbyrdaquent** [comment]
Conflicts resolved. Thank you.

**Comment by dbyrdaquent** [comment]
Sorry, did not intend to close PR.
@DaniHalfin

**Pull request merged**